xbmc/rendering/gl/CMakeLists.txt

```diff
@@ -8,4 +8,9 @@ set(HEADERS GUIWindowTestPatternGL.h
             ../MatrixGL.h
             GLShader.h)
 
+if(ARCH MATCHES arm)
```

Comment:
`if(ARCH MATCHES arm AND ENABLE_NEON)`

Reply:
If `ENABLE_NEON` isn't true the C(++) define `HAS_NEON` will also not be set. If that isn't set Matrix4Mul will never be called.

https://github.com/xbmc/xbmc/blob/e97f19f0bd0987224e37a0ad70f76362ec71f611/cmake/scripts/common/ArchSetup.cmake#L147-L149
https://github.com/xbmc/xbmc/blob/4398f988e1577495ab6a9ba1e43e6bd2ac65f465/xbmc/rendering/MatrixGL.cpp#L174-L177

Reply:
I confirm that I have -DHAS_NEON in my environment, but by chance, the NEON_FLAGS isn't defined, so I can set the one appropriate for my build. (that I expect will work for any build but might be less optimized than some others options).
So right now I'm (testing) adding -mfpu=neon unless NEON_FLAGS is already set.